airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/source.py

```diff
@@ -95,7 +95,10 @@ def check_connection(self, logger: logging.Logger, config: Mapping[str, Any]) ->
             if config.start_date and config.end_date < config.start_date:
                 return False, "End date must be equal or after start date."
 
-            api = API(access_token=config.access_token, page_size=config.page_size)
+            if config.credentials.auth_type == 'Client':
```

Comment:
Just small suggestion is to use maybe Enum class to represent the auth type value.